<launch>
  <!-- 定義bag_file參數，並設置預設值 -->
  <arg name="bag_file" default="$(find calibration)/raw_data/bag/corridor_normal/bag_20230829_185234.bag"/>

  <!-- 播放bag檔案 -->
  <node name="play_bag" pkg="rosbag" type="play" args="-l $(arg bag_file)"/>

  <!-- 執行Python腳本 -->
  <node name="save_files" pkg="calibration" type="bag_to_img_pcd.py" output="screen"/>
</launch>



